### ISSUE TITLE
[Request] Author field

### ISSUE THREAD

**Issue by milkyjoe90** [microfeed/microfeed#80]
First I wanted to say this is a wonderfully creative and awesome use of Cloudflare's tech, well done!

Now on to the feature request - would it be possible to add a field to the item which either takes the user's details from Cloudflare Access, or, as a stretch request, have an Author schema which can be tied to an item, and in the schema give the ability to upload an avatar/profile pic for the author so that the metadata of the post can be enriched?

Once again, thank you very much!

**Comment by kenara** [comment]
Right, the author info isn't associated with each item, just with the channel itself. It's assumed that the Channel operator is the author, which is the stated purpose of the software. If you open the little right-arrow thingy on the Edit Channel page, you'll see `{ "authors": [{"name": "Publisher Here"}] }` 

In my case there are content types that will probably never be supported and @wenbinf has explained somewhere that a multi-user environment such as I need is not on the roadmap. The project looks good and I'll let the author and contributors keep improving it.
